### ISSUE TITLE
detect_crop out of range error when using --merge

### ISSUE THREAD

**Issue by MelvinKool** [MelvinKool/TUEDownloader#5]
When I'm using the --merge option, I get an index out of range error.
This has to be triaged.

```
Traceback (most recent call last):
  File "/usr/local/bin/tuedownloader", line 11, in <module>
    load_entry_point('TUEDownloader==0.1', 'console_scripts', 'tuedownloader')()
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/download.py", line 434, in main
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/download.py", line 394, in download_channel
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/download.py", line 264, in download_video_showcase
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/editor.py", line 68, in diagonal
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/editor.py", line 28, in detect_crop
IndexError: list index out of range
```

**Comment by RickdeJager** [comment]
As far as I can tell this happens when the first second of the video is a black screen, for example if the first few minutes of the screencast is empty.  
  
See [this line](https://github.com/MelvinKool/TUEDownloader/blob/master/tuedownloader/editor.py#L18) for the timing settings.  
I picked 1 second because thats _usually_ enough to get a crop, and this filter takes quite a long time if you run it on the full video.
  
Anyways, I could change it to fall back to black bars / no crop if the regex doesn't find anything.

**Comment by MelvinKool** [comment]
@RickdeJager could you take a look at cb32cf24c4302cf4f209854f5dd001cc58999af9?
In this commit I've removed the subprocess command and changed it to the python-ffmpeg alternative.
Maybe we could add an extra sanity check for the dimensions as well.

If you think my commit is good, I'll merge it with master.
The only thing I'm wondering is wheter -vframes 2 is always enough.

**Comment by RickdeJager** [comment]
Could you send then output to `/dev/null` (linux) or `NUL` (windows) directly? Other than that, I think -vframes 2 is rather low, download/merge will take quite some time anyways so taking a few extra seconds to determine crop is fine imo. (vframes 20-ish?)
  
As for the cropdetect bug, I've "solved" it on my fork by returning the original dimensions if cropdetect fails. [See this branch](https://github.com/RickdeJager/TUEDownloader/tree/bugfix/graceful-crop-fail).

**Comment by MelvinKool** [comment]
Awesome, I was thinking about fixing it, but you already have it done hahah.
I'll include it in the develop branch.

While I was testing yesterday, /dev/null was not accepted by ffmpeg, since it tried to detect file format.
I'll increase -vframes to 20, sounds like a safer option.

**Comment by MelvinKool** [comment]
Fixed in bd9fcb449d3b0fea7837c4901eaf1f3b2cfca02e
